conf/squid/rados/stretch-mode-host-location-attrs.yaml

```diff
@@ -70,8 +70,5 @@ globals:
         no-of-volumes: 4
         disk-size: 15
       node8:
-        image-name:
```

Comment:
instead of removal, can we comment it for now?

Reply:
done